docs/dev/development-process/release-new-version.md

```diff
@@ -1,8 +1,37 @@
-To release a new version:
-
-1. Write up a changelog of all the changes made between the current version and the new version. You can run 
-`git log --merges --grep 'Merge pull request' --pretty=format:%s [old-version]..head`
- to get every PR made between the previuous version and now
-2. Update `VERSION` in `posthog/settings.py` AND `posthog-production/settings.py`
-3. `git tag -a [version] -m "Version [version]"`
-4. `git push origin head --tags`
+# Releasing a new version
+
+At the moment, we release a new version weekly. This might change in the future.
+
+## Version numbers
+
+Every week we up the 'minor' in `major.minor.patch`. At the moment, we're at version 1 for major. This will only change once we have released sufficient functionality under stage 2 of [our roadmap](https://docs.posthog.com/#/roadmap).
+
+Hopefully we will not have to do many patch versions, but if between releasing version we discover a breaking bug, we will.
+
+
+## Timeline
+
+We release every Wednesday. We have no set time for this but it ends up being afternoon/evening UK time by the time we have everything ready.
```

Comment:
Do we know where a majority of our users are? If there's no clustering then it's quite useless, but pushing an update morning time Tuesday/Wednesday (which might be coincidentally happening now) seems optimal just from intuition because people will see the update in email and be able to react to it during the day. Otherwise, the process makes sense to me!

Reply:
Good question and the honest answer is [I'm not sure](https://github.com/PostHog/posthog/issues/191) but agree we should aim for afternoon UK time, which will be late morning eastern and early morning pacific time.